scripts/style_check.sh

```diff
@@ -235,6 +235,24 @@ sev[47]=MAYOR; cat[47]='Extra space before )'
 sev[48]=NONE;  cat[48]="LAST"
 
 
+
+# -----------------------------------------------------------------------------
+#
+# Exceptions per library
+#
+# o Rule 6 for mongoBackend
```

Comment:
Rule 6... This is an internal numbering you use for modifications in the script? Or does it correspond with some Mx or Sx in our rules set for coding style?